pkg/common/common.go

```diff
@@ -65,8 +64,7 @@ func ReplaceManifestUnSupportedAPIs(origManifest, mapFile string, kubeConfig Kub
 	if err != nil {
 		return "", err
 	}
-	kubeVersion, err := strconv.ParseFloat(kubeVersionStr, 32)
-	if err != nil {
+	if !semver.IsValid(kubeVersionStr) {
 		errors.Wrap(err, "Failed to get Kubernetes server version")
```

Comment:
errors.Wrap(err, "Failed to get Kubernetes server version") --> errors.Errorf("Failed to get Kubernetes server version")